src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -39,11 +39,11 @@ public class Transaction : IEquatable<Transaction>, IInventory, IInteroperable
         private TransactionAttribute[] attributes;
```

Comment:
Is one attribute enough? @erikzhang what do you think?

Reply:
I think that if we have attributes, it should be an array.